astroplan/data/observatories.json

```diff
@@ -381,4 +381,18 @@
         "name": "mro",
         "source": "MRO website"
     }
-}
\ No newline at end of file
+    "SALT": {
+        "source" : "SALT website",
+        "elevation": 1700, 
+        "name": "South African Large Telescope", 
+        "longitude_unit": "degree", 
+        "latitude_unit": "degree", 
+        "latitude": 20.810499, 
+        "elevation_unit": "meter", 
+        "longitude": 32.561633, 
```

Comment:
Are they? Shouldn't they be the other way around, following most geography standards?